config.c

```diff
@@ -306,6 +306,9 @@ void read_FTLconf(void)
 	// AUDITLISTFILE
 	getpath(fp, "AUDITLISTFILE", "/etc/pihole/auditlog.list", &files.auditlist);
 
+	// GRAVITYFILE
+	getpath(fp, "GRAVITYFILE", "/etc/pihole/gravity.db", &FTLfiles.gravitydb);
```

Comment:
GRAVITYFILE is used above for `/etc/pihole/gravity.list`. A new setting must be used, such as `GRAVITYDB`.